operations/node_management/Update_the_HPE_Node_BIOS_Time.md

```diff
@@ -0,0 +1,71 @@
+# Update the HPE Node BIOS Time
+
+Check and set the time for HPE nodes.
+
+If the console log indicates the time between the rest of the system and the node is off by several hours, it prevents the spire-agent from getting a valid certificate, causing the node boot to drop into the dracut emergency shell.
+
+## Procedure
+
+1. Log in to a second terminal session in order to watch the node's console.
+
+    ***Please open this link in a new tab or page*** [Log in to a Node Using ConMan](../conman/Log_in_to_a_Node_Using_ConMan.md)
+
+    The first terminal session will be needed to run the commands to boot the node into the BIOS menu.
+
+1. (`ncn#`) In another terminal session set the following environment variables:
+
+    1. The BMC hostname.
+
+        ```bash
+        BMC=x3000c0s3b0
+        ```
+
+    1. Export the `root` user password of the BMC.
+
+        > NOTE: `read -s` is used to prevent the password from echoing to the screen or
+        > being saved in the shell history.
+
+        ```bash
+        read -r -s -p "${BMC} root password: " IPMI_PASSWORD
+        export IPMI_PASSWORD
+        ```
+
+1. (`ncn#`) Boot the node into its BIOS via the serial console.
+
+    When `ipmitool` is used to boot a HPE node into its BIOS menu it will not be available on on the nodes serial console, due to the node booting into a graphical BIOS menu. To access the serial version of the BIOS setup. Perform the `ipmitool` steps
+    above to boot the node. Then in conman press `ESC+9` key combination when you see the following messages in the console. This will open a menu you use to enter the BIOS via serial console.
+
+    ```text
+    For access via BIOS Serial Console:
+    Press 'ESC+9' for System Utilities
+    Press 'ESC+0' for Intelligent Provisioning
+    Press 'ESC+!' for One-Time Boot Menu
+    Press 'ESC+@' for Network Boot
+    ```
+
+    ```bash
```

Comment:
Earlier the text refers to the "above" ipmitool commands. Should these be above that?

Reply:
I've changed the wording to the following: https://github.com/Cray-HPE/docs-csm/pull/2344/commits/d95c1e8bf36de5c16447a201cb2ecf3e6f4b5113